### ISSUE TITLE
Add location and interfaces details to the nodes endpoint

### ISSUE THREAD

**Issue by AhmedHanafy725** [threefoldtech/tfgridclient_proxy#132]

**Comment by sameh-farouk** [comment]
**Update:**
PR ready to review https://github.com/threefoldtech/tfgridclient_proxy/pull/242

**Comment by A-Harby** [comment]
Tried to get node longitude & latitude of a many different node through the endpoint e.g. `https://gridproxy.dev.grid.tf/nodes/19` , but all values appear to be null.
despite all values are retrieved and displayed on dashboard.
![image](https://user-images.githubusercontent.com/109473873/209982897-05c42603-361b-403e-876f-fa75dce958dc.png)

**Comment by sameh-farouk** [comment]
@A-Harby thanks for reporting this. Recently we added long and lat to API response, due to having two different types for node, `types.Node` and `types.NodeWithNestedCapacity` some endpoints return a response using the first type, like `/nodes`, and another returns a response using the second type, like `/nodes/{id}`, in the recent commit, we filled only the first type, `types.Node` with this data, and overlooked updating the other model `types.NodeWithNestedCapacity`.

I pushed a PR that should when gets merged and released handle filling the data to the second type as well. 

![Screenshot from 2022-12-29 20-45-49](https://user-images.githubusercontent.com/42457449/209995936-3d16ce66-3264-4f3c-abfc-05cc06b12202.png)
